packages/neovim/README.md

```diff
@@ -0,0 +1,195 @@
+# neovim-client
+
+| CI (Linux, macOS, Windows) | Coverage | npm | Gitter |
+|----------------------------|----------|-----|--------|
+| [![ci](https://github.com/neovim/node-client/actions/workflows/ci.yml/badge.svg)](https://github.com/neovim/node-client/actions/workflows/ci.yml) | [![Coverage Badge][]][Coverage Report] | [![npm version][]][npm package] | [![Gitter Badge][]][Gitter] |
+
+Currently tested for node >= 10
+
+## Installation
+Install the `neovim` package globally using `npm`.
+
+```sh
+npm install -g neovim
+```
+
+A global package is required for neovim to be able to communicate with a plugin.
+
+## Usage
+This package exports a single `attach()` function which takes a pair of
+write/read streams and invokes a callback with a Nvim API object.
+
+### `attach`
```

Comment:
are we repeating info from the main readme here? is this for https://www.npmjs.com/package/neovim I guess?

can we link to this readme from the main readme, to avoid duplicating the info